starterkits/saas/src/app/(app)/(user)/org/members/invite/_components/send-invite-link.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { sendOrgInviteEmail } from "@/server/actions/send-org-invite-email";
+import { useEffect, useState } from "react";
+import { toast } from "sonner";
+
+
+type SendInviteLinkProps = {
+    inviteLink: string;
+    orgName: string;
+};
+
+export function SendInviteLink({ inviteLink, orgName }: SendInviteLinkProps) {
+    const [isPending, setIsPending] = useState(false);
+    const [isSent, setIsSent] = useState(false);
+    const [sendInviteEmail, setSendInviteEmail] = useState("");
+
+    const sendInvite = async () => {
+        setIsPending(true);
+        const sendInvitePromise = () => sendOrgInviteEmail({
+            email: sendInviteEmail,
+            orgName: orgName,
+            invLink: inviteLink
+        });
+
+        toast.promise(sendInvitePromise(), {
+            loading: 'Sending invite...',
+            success: () => {
+                setIsSent(true);
+                return "Invite sent";
+            },
+            error: 'Error sending invite',
+            finally() {
+                setIsPending(false);
+            },
+        });
+    };
+
+    useEffect(() => {
+        let timeout: NodeJS.Timeout;
+
+        if (isSent) {
+            timeout = setTimeout(() => {
+                setIsSent(false);
+            }, 3000);
+        }
+
+        return () => {
+            clearTimeout(timeout);
+        };
+    }, [isSent]);
+
+    return (
+        <div className="flex gap-4 md:flex-row md:items-center md:justify-between">
```

Comment:
Can you please use shadcn/ui form for better accessibility. for eg submit by pressing enter key, and zod email validation, currently I can send invite on non email input aswell for eg: "someword". refer others forms in codebase if you want some help or reach me out